campaigns.json

```diff
@@ -3176,13 +3255,6 @@
        "T2_CH_CERN_HLT"
     ],
     "secondaries": {
-      "/Neutrino_E-10_gun/RunIIFall17FSPrePremix-PUFSUL16_106X_mcRun2_asymptotic_v16-v1/PREMIX" :
```

Comment:
Why do you remove this secondary? @scarletnorberg

Reply:
Because it was not needed

Reply:
The other FS campaign had two although I do not think both are needed. Jordan confirmed that only one is needed on the JIRA ticket.